### ISSUE TITLE
Issue after update v6.2.0

### ISSUE THREAD

**Issue by naudski** [123Venom/plugin.video.venom#121]
I'm having some issues with my liked lists after the latest update. My liked lists sections are empty and the finish watching section in my movies section does not work anymore. After reverting back to v6.1.9 everything is working again. This issue occured on both my kodi devices. Any ideas?

**Comment by 123Venom** [comment]
You're now the second one reporting this issue with liked lists not loading..sadly I've not been able to duplicate it myself.  What we did as a fast cure for him was deleting the traktsync.db file in userdata plugin.video.venom folder and let it rebuild itself.  I've got to poke about more into this for a solution.  I may have have to add a new function to rebuild the db from scratch. For some reason the new table structure worked sometimes and sometimes does not...haven't put my finger on the cause yet.
